parameters/world_modeling/world_model_demo_intern.yaml

```diff
@@ -56,12 +56,6 @@ plugins:
           min_distance: 0.4
           max_distance: 2.0
 
```

Comment:
Do we not require the people detector in demo_intern?

Reply:
Not at the moment.